infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -145,17 +335,20 @@ def restore_database(backup_fpath):
 
 
 def main():
+    print("System args:", sys.argv)
     print("Running database-backup process with subcommand", sys.argv[1])
     if sys.argv[1] == "backup":
-        backup_database(make_backup_fpath())
+        s3_backup_object_name = make_backup_object_name_from_datetime()
+        backup_database(s3_backup_object_name)
```

Comment:
```suggestion
        backup_database(backup_object_name=s3_backup_object_name)
```

Unlabeled argument :(